src/series.vala

```diff
@@ -2,23 +2,27 @@ using Gee;
 namespace LiveChart { 
 
     public class Series : Object {
-
+        private Gee.Map<Serie, ulong> signals = new Gee.HashMap<Serie, ulong>();
         private Gee.ArrayList<Serie> series = new Gee.ArrayList<Serie>();
-        private Chart chart;
+        private weak Chart chart;
```

Comment:
👍

Reply:
`unowned` is meaningful than `weak`, I like using it more than `weak`, :-)

Reply:
https://wiki.gnome.org/Projects/Vala/ReferenceHandling#Unowned_References

> However, you should use weak only for breaking reference cycles and unowned only for ownership issues as described obove.

I don't know which I should to do... ;(
(Yes, I know `unowned` is more meaningful, so in my old code, I used `unowned`).